test/e2e/silo-access.e2e.ts

```diff
@@ -67,7 +67,7 @@ test('Click through silo access page', async ({ page }) => {
     .click()
   await page.click('role=menuitem[name="Change role"]')
 
-  await expectVisible(page, ['role=heading[name*="Change user role"]'])
+  await expectVisible(page, ['role=heading[name*="Edit role"]'])
```

Comment:
“Change user role” is a little more descriptive? I could go either way. Also if it shows up for editing a group role too it’s misleading.

Reply:
Yeah, I agree that `Change` is probably more accurate than `Edit`, as they aren't editing the _role_, per se. `Change permissions`?

Reply:
On the actions modal we say `Change role`, so I think that makes sense. Should we determine whether it's a `user` or `group` and pass in the appropriate term? `Change user role` / `Change group role`

Reply:
Permissions isn’t a term in the API, so I prefer to avoid it. Really what they’re doing in this case is updating the _policy_ on the resource, and the UI is sort of deliberately obscuring that (uncharacteristically) because it is so much more natural to think about operating on the user.

Reply:
Hold the phone. There was a TODO in the `project-access` and `silo-access` files noting that we should `// TODO: show user name in header or SOMEWHERE` … this seems like as good a time as any to update that in the modal's title.

Reply:
<img width="515" alt="Screenshot 2024-03-08 at 5 42 00 PM" src="https://github.com/oxidecomputer/console/assets/22547/8bc7b8b4-c1d4-4995-8734-b7c6680c2e83">

This copy feels a little clunky to me, but dynamic headers might just be like that.

Reply:
Yea, it’s a bit weird but it’s still an improvement.

Reply:
Though it worked if the name was hypenated, it overran the boundary without hyphens …
<img width="511" alt="Screenshot 2024-03-11 at 10 39 13 AM" src="https://github.com/oxidecomputer/console/assets/22547/b249a715-0834-4617-96be-f0daa1ac2963">
… so I added `word-break: break-all` to the class for that header:
<img width="514" alt="Screenshot 2024-03-11 at 10 50 42 AM" src="https://github.com/oxidecomputer/console/assets/22547/12591fd2-b5e4-47a0-9a4c-ac58f644acf0">

Reply:
Though …
<img width="495" alt="Screenshot 2024-03-11 at 11 19 18 AM" src="https://github.com/oxidecomputer/console/assets/22547/b718dbd8-20ea-42f2-9a9f-f522b90048a3">


I have an idea I'll try here.

Reply:
what about `break-words` instead of `break-all`?

Reply:
I tried that, plus wrapping a container in `display: inline-block` and individual words in both `break-words` and `break-all`, but neither worked as desired 100% of the time. I think users will run into more situations with something like `Name ofPerson` or `hyphenated-group-name` as opposed to `unhyphenatedLongGroupName`, so `break-words` is probably the best solution for now.